--- eventlet-0.25.1/eventlet/__init__.py.orig	2019-08-21 23:29:28.000000000 +0200
+++ eventlet-0.25.1/eventlet/__init__.py	2020-01-07 05:30:11.537444971 +0100
@@ -21,8 +21,11 @@
     # Helpful when CPython < 3.5 on Linux blocked in `os.waitpid(-1)` before first use of hub.
     # Example: gunicorn
     # https://github.com/eventlet/eventlet/issues/401#issuecomment-327500352
-    import monotonic
-    del monotonic
+    try:
+        import monotonic
+        del monotonic
+    except ImportError:
+        pass
 
     connect = convenience.connect
     listen = convenience.listen
--- eventlet-0.25.1/eventlet/hubs/hub.py.orig	2019-05-24 11:00:38.000000000 +0200
+++ eventlet-0.25.1/eventlet/hubs/hub.py	2020-01-07 05:30:59.527184988 +0100
@@ -22,7 +22,10 @@
 import eventlet.hubs
 from eventlet.hubs import timer
 from eventlet.support import greenlets as greenlet, clear_sys_exc_info
-import monotonic
+try:
+    from monotonic import monotonic
+except ImportError:
+    from time import monotonic
 import six
 
 g_prevent_multiple_readers = True
@@ -120,7 +123,7 @@
         self.closed = []
 
         if clock is None:
-            clock = monotonic.monotonic
+            clock = monotonic
         self.clock = clock
 
         self.greenlet = greenlet.greenlet(self.run)
--- eventlet-0.25.1/setup.py.orig	2020-01-07 05:31:16.320427345 +0100
+++ eventlet-0.25.1/setup.py	2020-01-07 05:31:19.623742782 +0100
@@ -18,7 +18,7 @@
         'dnspython >= 1.15.0',
         'enum34;python_version<"3.4"',
         'greenlet >= 0.3',
-        'monotonic >= 1.4',
+        'monotonic >= 1.4;python_version<"3.3"',
         'six >= 1.10.0',
     ),
     zip_safe=False,
